opsimulator/opsimulator.go

```diff
@@ -155,24 +134,6 @@ func (opSim *OpSimulator) EthClient() *ethclient.Client {
 	return opSim.ethClient
 }
 
-func (opSim *OpSimulator) startStartupTasks(ctx context.Context) {
-	opSim.startupTasks.Go(func() error {
```

Comment:
does removing this go routine slowdown startup time since now we will not be configuring interop for each chain in parallel?

Reply:
per my earlier comment they weren't being init'd in parallel

Reply:
Let me also make this concurrent in the orchestrator

Reply:
okay pushed, startup time is back to being fast again